spec/recurly/account_spec.rb

```diff
@@ -433,6 +433,17 @@
     end
   end
 
+  describe "override_business_entity_id" do
+    let(:account) {
+      stub_api_request :get, "accounts/abcdef1234567890", "accounts/show-200"
+      Account.find("abcdef1234567890")
+    }
+
+    it "should have an override_business_entity_id" do
+      account.override_business_entity_id.must_equal("sbup2j0fx800")
+    end
```

Comment:
For this and invoice_spec `account.business_entity.must_be_instance_of(BusinessEntity)` and `invoice.business_entity.must_be_instance_of(BusinessEntity)` might be more meaningful specs, since they show it has the associated object, and the `business_entity` behavior is specced in `business_entity_spec`, what do you think?

Reply:
That makes sense. I can make those changes